packages/plugin-table/test/tableTestUtils.ts

```diff
@@ -36,8 +36,7 @@ export function testRowspan(cells: TableCellNode[], expected: number[]): void {
 export function testStyles(cells: TableCellNode[], expected: string[]): void {
     for (let i = 0; i < cells.length; i += 1) {
         const desc = `${i}th cell preserved style`;
-        const attributes = cells[i].modifiers.find(Attributes);
-        expect(attributes?.get('style')).to.equal(expected[i], desc);
+        expect(cells[i].modifiers.get(Attributes).get('style')).to.equal(expected[i], desc);
```

Comment:
It's a bit irrelevant because it's a test but technically this should be `find` as well.